### DISCUSSION TITLE
markdown question

### DISCUSSION THREAD

**Discussion by ktenney** [TriliumNext/Trilium#1401] [General]
I'm having trouble with words containing multiple underscores.
Is there a way to prevent them going italic / bold?

Thanks,
Kent

**Comment by zadam** [comment]
This is called autoformatting, if the editor autoformats incorrectly, press escape and it will revert back to unformatted text.

**Reply by ktenney** [reply]
when I type __main__ it renders as bold.
I'm unable to use the escape key to remove the formatting.
Specifically how is that done?

Thanks,
Kent

**Reply by ktenney** [reply]
arghh
what I typed was underscoreunderscoremainunserscoreunderscore
:=]

**Reply by zadam** [reply]
Sorry, I meant CTRL-Z (undo), not Escape.

![Peek 2020-11-10 22-33](https://user-images.githubusercontent.com/617641/98736216-d29c4f00-23a4-11eb-92f0-45032757f9fc.gif)

**Comment by ktenney** [comment]
Yay!
Thanks,
Kent
